src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseMultiSignMessage.cs

```diff
@@ -33,6 +41,37 @@ internal CoseMultiSignMessage(CoseHeaderMap protectedHeader, CoseHeaderMap unpro
             _signatures = signatures;
         }
 
+        /// <summary>
+        /// Signs the specified content and encodes it as a COSE_Sign message with detached content.
+        /// </summary>
+        /// <param name="detachedContent">The content to sign.</param>
+        /// <param name="signer">The signer information used to sign <paramref name="detachedContent"/>.</param>
+        /// <param name="protectedHeaders">The protected header parameters to append to the message's content layer.</param>
+        /// <param name="unprotectedHeaders">The unprotected header parameters to append to the message's content layer.</param>
+        /// <param name="associatedData">The extra data associated with the signature, which must also be provided during verification.</param>
+        /// <returns>The encoded message.</returns>
+        /// <exception cref="ArgumentNullException">
+        ///   <para>
+        ///     <paramref name="detachedContent"/> is <see langword="null"/>.
+        ///   </para>
+        ///   <para>-or-</para>
+        ///   <para>
+        ///     <paramref name="signer"/> is <see langword="null"/>.
+        ///   </para>
+        /// </exception>
+        /// <exception cref="ArgumentException">
+        ///   <para>
+        ///     The <paramref name="protectedHeaders" /> and <paramref name="unprotectedHeaders" /> collections have one or more labels in common.
+        ///   </para>
+        ///   <para>-or-</para>
+        ///   <para>
+        ///     The <see cref="CoseSigner.ProtectedHeaders"/> and <see cref="CoseSigner.UnprotectedHeaders"/> collections in <paramref name="signer"/> have one or more labels in common.
+        ///   </para>
+        ///   <para>-or-</para>
+        ///   <para>
+        ///     One of the labels specified in a <see cref="CoseHeaderLabel.CriticalHeaders"/> header is missing.
```

Comment:
If we wanted to be pedantic

```suggestion
        ///     One or more of the labels specified in a <see cref="CoseHeaderLabel.CriticalHeaders"/> header is missing.
```